kubetest/e2e.go

```diff
@@ -508,6 +508,11 @@ func nodeTest(testArgs, nodeArgs []string, nodeTestArgs, project, zone string) e
 		log.Printf("cwd : %s", wd)
 	}
 
+	sshKeyPath := os.Getenv("JENKINS_GCE_SSH_PRIVATE_KEY_FILE")
+	if _, err := os.Stat(sshKeyPath); err != nil {
```

Comment:
Use `os.IsNotExist` to check or just print the `err` to make sure it's the error we expected?

Reply:
Otherwise LGTM.

Reply:
printed the error